repo_tools_data_schema/repo_tools_data_schema.py

```diff
@@ -82,10 +82,11 @@ def not_data_key(s):
 
 
 def one_of_keys(*keys):
+    """Checks that at least one key is present (not exclusive OR)"""
```

Comment:
Thanks for adding the docstring. I was embarrassed to see that I hadn't written one.